charts/argo-cd/values.yaml

```diff
@@ -1139,9 +1160,78 @@ server:
   # -- [General Argo CD configuration]
   # @default -- See [values.yaml]
   config:
+    resource.inclusions: |
+      - apiGroups:
+        - "networking.k8s.io"
+        kinds:
+        - Ingress
+        - NetworkPolicy
+        clusters:
+        - https://10.135.192.1:443
+        - https://kubernetes.default.svc
+      - apiGroups:
+        - "cert-manager.io"
+        kinds:
+        - Certificate
+        clusters:
+        - https://10.135.192.1:443
+        - https://kubernetes.default.svc
```

Comment:
Same here

Reply:
This was the format given by @YitzyD so I assumed there was a reason?

Reply:
To be honest, when I threw this together, one didn't work, then one did, and I forgot to remove the one that didn't :facepalm:. I guess test functionality with one or the other? They may both work now, in which case the dns name is enough.

Reply:
Well I tried switching to just one or the other, and it stayed at an Unknown status, then switched back to both and still Unknown status..so it's not picking up the cluster status anymore which is odd. having to look into it.

Reply:
Ok figured it out, it's the IP not dns that works. removing the dns one.